backend/src/components/user.js

```diff
@@ -264,8 +264,15 @@ function mapContactForBack(data) {
   const contact = new MappableObjectForBack(data, ContactMappings).toJSON()
   contact.entityNameSet = 'contacts'
   contact.actionMode = 'Update'
-  contact['ofm_portal_role_id@odata.bind'] = `/ofm_portal_roles(${data?.role?.roleId})`
+
+  // Handle role Update/Delete
+  if (contact.role) {
+    contact['ofm_portal_role_id@odata.bind'] = `/ofm_portal_roles(${contact.role.roleId})`
+  } else {
```

Comment:
Is this else statement necessary? should this if else statement be like:
```
if (contact.role) {
    contact['ofm_portal_role_id@odata.bind'] = `/ofm_portal_roles(${contact.role.roleId})`
    delete contact.role
  }
```

Reply:
There's different syntax to remove the role.